### PR TITLE
Storage of Image as Class Variable

### LINKED ISSUE #52: Store image array as a class variables
Up to now, we tried to store processed images as image files in folders, 

This issue focuses on creating a class variable. 
All the images would be stored in the class variable and also folders as input by the user.

### PR DESCRIPTION
**Pull request by Saptar001**
# Storage of Image as Class Variable:

The issue was on storing images as class variable. 
Here what I did is created an empty list, appended the processed images in the list and converted the list into a 3D array using np.stack(list)

Fixes #52 

## Type of change:

- [x] New feature (non-breaking change which adds functionality)

 
# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Saptar001** [comment]
Hello @chebroluharika , @yvkrishna  I made the relevant asked changes and made the PR, in the Filters.py inside the "applyMedian" function as per said by @yvkrishna .
Do go through my work and give your reviews .

Thank You :)

**Comment by chebroluharika** [comment]
@Saptar001 I think you should include this change in DataLoader file too (wherever we are storing images in form of numpy arrays, you should incorporate this)

**Comment by chebroluharika** [comment]
@yvkrishna here my opinion is to implement this change at every module in filters.py
Could you please respond if thats right approach

**Comment by yvkrishna** [comment]
> @Saptar001 I think you should include this change in DataLoader file too (wherever we are storing images in form of numpy arrays, you should incorporate this)

Yes

**Comment by yvkrishna** [comment]
> Hello @chebroluharika , @yvkrishna I made the relevant asked changes and made the PR, in the Filters.py inside the "applyMedian" function as per said by @yvkrishna .
> Do go through my work and give your reviews .
> 
> Thank You :)

I believe that it creates an empty list each time while it is in the loop. And it is also not a class variable. @chebroluharika can you check it once

**Comment by yvkrishna** [comment]
> @yvkrishna here my opinion is to implement this change at every module in filters.py
> Could you please respond if thats right approach

Yeah, we need to do that, but if this part in median filter is validated, we can make the same changes to all the functions in filters.

File: flow2ml/Filters.py [hunk 1]
```diff
@@ -37,6 +37,12 @@ def applyMedian(self,classPath):
       if img is not None:
         # applies median filter to the image.
         median = cv2.medianBlur(img, 5)
+
+        lt=[] #empty list
+
+        lt.append(median) # storing the images in the list
+
+        c=np.stack(lt) # converting the stored iamges in the list into a 3D array which is "c" here
```
Comment (chebroluharika) [inline flow2ml/Filters.py:45]: You must declare variable c as class variable and proceed. Store all processes images into variable “c”
And also give a meaningful name for variable “c”

**Comment by chebroluharika** [comment]
Closing this as we didn’t receive any update from long back

**Pull request closed without merging**